src/main/kotlin/com/cognifide/gradle/aem/deploy/tasks/AbstractTask.kt

```diff
@@ -115,19 +122,19 @@ abstract class AbstractTask : DefaultTask(), AemTask {
             val response = InstallResponse(json)
 
             when (response.status) {
-                InstallResponse.Status.SUCCESS -> if (response.errors.isEmpty()) {
+                AbstractHtmlResponse.Status.SUCCESS -> if (response.errors.isEmpty()) {
```

Comment:
agreed to leave as it is, with AbstractHtmlResponse name changed to HtmlResponse